sorted-r-blacklist

```diff
@@ -58,20 +58,16 @@ recipes/bioconductor-bubbletree
 # TODO: bioconductor-cexor needs specific version of genomicranges?
 #recipes/bioconductor-cexor
 
-
+# conflicts with recent Rcpp version
+recipes/r-histonehmm
```

Comment:
the main problem is that I have no idea which version of Rcpp it needs, the dev site is not explicit either..